drivers/hmis/app/graphql/schema.graphql

```diff
@@ -8602,12 +8602,13 @@ type ProjectAccess {
   canManageDeniedReferrals: Boolean!
   canManageExternalFormSubmissions: Boolean!
   canManageIncomingReferrals: Boolean!
-  canManageInventory: Boolean!
```

Comment:
This will be a breaking change: if the frontend is on an older version that is trying to fetch canManageInventory, and then we deploy this, that request will raise an exception. My suggestion on the frontend PR was to stop resolving this field, so that we can safely remove it in a future release. Sorry for the confusion!
As of yet the only documentation we have around this is in the Runbook, but maybe it should be moved to github somewhere and expanded upon.

Reply:
Thanks Gig for cleaning this up - I was rushing and didn't think it through. Thanks for the fix!

Reply:
@gigxz let's look at this [build fail](https://github.com/greenriver/hmis-frontend/actions/runs/11470351871) introduced here together tomorrow. I spent some time today puzzling over [PermissionFilter here](https://github.com/greenriver/hmis-frontend/blob/release-139/src/modules/permissions/PermissionsFilters.tsx), but wasn't quite able to figure it out.

It is the same build fail I encountered [a couple commits ago](https://github.com/greenriver/hmis-frontend/pull/963/commits/346d6f2d75f159766c4c6eac66c73f53548d0a7e), which I attempted to fix in my next commit by just [ripping out `canManageInventory` entirely](https://github.com/greenriver/hmis-frontend/pull/963/commits/8846cc862513a979075cfddd007e5fbb84708cb2). You rightly pointed out that that's not backwards compatible, and needs to be reverted, but the build fail reappears.

We could just `ts-ignore` it - maybe that's the best solution?

Reply:
paper-trail - opened https://github.com/greenriver/hmis-frontend/pull/970